test/colab/test.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -eo pipefail
+
+trap 'wait;' SIGINT SIGTERM
```

Comment:
Trap mechanism to ensure that interruptions stop the docker run as soon as it's received.